action.yml

```diff
@@ -24,7 +24,7 @@ inputs:
     required: false
   runtime:
     description: 'Runtime of the application, note: AppRunner only supports `NODEJS_12` and `PYTHON_3` runtime types'
-    required: true
```

Comment:
Is this made optional to support docker image based deployments?

Reply:
Yes, runtime is only required when deploying from source. This is validated elsewhere :+1: